monai/transforms/transform.py

```diff
@@ -183,9 +187,17 @@ class Randomizable(ThreadUnsafe, RandomizableTrait):
     the random states will be duplicated.
     """
 
-    R: np.random.RandomState = np.random.RandomState()
-
-    def set_random_state(self, seed: int | None = None, state: np.random.RandomState | None = None) -> Randomizable:
+    R: SupportsRandomGeneration = LegacyRandomStateAdaptor()  # FIXME Why is this initialized here?
```

Comment:
Seems we also need to update the docstring in L181 to tell users we may change to Generator near future.